src/CakeAdapter.php

```diff
@@ -344,15 +344,15 @@ public function fetchAll($sql)
     }
 
     /**
-     * Inserts data into the table
+     * Inserts data into a table.
      *
      * @param Table $table where to insert data
-     * @param array $columns column names
-     * @param $data
+     * @param array $row
+     * @return void
```

Comment:
@dereuromark okay noted moving forwards, however I'm not going to change it now as unfortunately there are quite a few references.